src/order/pfilter.lean

```diff
@@ -52,6 +63,9 @@ instance : has_coe (pfilter P) (set P) := ⟨λ F, F.dual.carrier⟩
 /-- For the notation `x ∈ F`. -/
 instance : has_mem P (pfilter P) := ⟨λ x F, x ∈ (F : set P)⟩
 
+lemma is_pfilter (F : pfilter P) : is_pfilter (F : set P) :=
+(pfilter.dual F).is_ideal
```

Comment:
Since `F` is a `pfilter` and `pfilter.dual` lives in the `pfilter` namespace, you can use `F.dual` directly:
```suggestion
F.dual.is_ideal
```